pytest_httpserver/httpserver.py

```diff
@@ -51,14 +53,44 @@ class RequestMatcher:
     :param query_string: the http query string starting with ``?``, such as ``?username=user``
     """
 
+    class HeaderValueMatcher:
```

Comment:
Could you move this class to the module level? I think having class definition in a class is very unusual in python and having a separate class would make clearer code.

Reply:
Done